File: Problem Solving/two_sum_leet_code.py
class Solution:
    def twoSum(self, a, target):
        maps={}
        count=0
        for i in range(len(a)):
            value=target-a[i]
            if value in maps:
                return [maps[value],i+1]
            else:
                maps[a[i]]=i+1

File: Problem Solving/two_sum_leet_code.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
	     HashMap<Integer,Integer> maps=new HashMap<>();
		for (int i=0;i<nums.length;i++)
		{
				value=target-nums[i];
				if(maps.get(value)!=null){

					return new int[] {maps.get(value),i}
				}
				else{
					maps.put(nums[i],i);
				}
		}        
    }
}